cmd/agent/app/flare.go

```diff
@@ -197,6 +197,10 @@ func makeFlare(caseID string) error {
 		return err
 	}
 
+	logLevel := config.Datadog.GetString("log_level")
```

Comment:
I think we can't just cleanly use the config string as the log level [can be dynamically set](https://github.com/DataDog/datadog-agent/blob/0454961e636342c9fbab9e561e6346ae804679a9/pkg/config/settings/runtime_setting_log_level.go#L14)

Reply:
@monganai I think you would want logic similar to [this](https://github.com/DataDog/datadog-agent/blob/main/pkg/collector/corechecks/snmp/internal/session/session.go#L127-L135) to get the right level. The `String()` on that level object should give you the string representation of it from what I've read in seelog docs.